content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -0,0 +1,156 @@
+| [Taints](/clusters/cluster-management/taints#overviewontaints) | Sets toleration to pods and allows (but does not require) the pods to schedule onto nodes with matching taints. |
+| Instance type | The compute instance type to be used for all nodes in the node pool. |
+| Availability Zones | These are multiple, isolated locations within a Region. When you choose one or more availability zones, Palette provides fault tolerance to guard against hardware failures and network failures by provisioning nodes across availability zones if multiple zones are selected. |
+| Disk Size | Specify the required storage size. |
+
+
+12. You can configure the following cluster management features now if needed, or you can do it later:
+
+    - Manage Machines
+    - Scan Policies
+    - Backup Policies
+    - Review settings and deploy the cluster. 
+
+
+13. Review settings and deploy the cluster. 
+
+
+## Validation
+
+You can validate your cluster is available by reviewing the cluster details page. Navigate to the left **Main Menu** and click **Clusters**. The **Clusters** page lists all available clusters Palette manages. Select the cluster to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
+
+
+
+# Delete a MAAS Cluster
+
+When you delete a MAAS cluster, all machines and associated storage disks that were created for the cluster are removed. 
+
+Follow these steps to delete a MAAS cluster.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and click **Clusters**. 
+
+
+3. Select the cluster you want to delete.
+
+
+4. Click the **Settings drop-down Menu**, and choose **Delete**.
+
+The cluster status is updated to **Deleting** while cluster resources are being deleted. When all resources are successfully deleted, the cluster status is updated to **Deleted** and the cluster is removed from the list. The delete operation returns the edge hosts to the **Ready** state. All the artifacts related to the Kubernetes distribution are removed.
+
+
+## Force Delete a Cluster
+
+If a cluster is stuck in the Deletion state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scopes. To force delete a cluster, follow the same steps outlined in [Delete a MAAS Cluster](/clusters/data-center/maas/create-manage-maas-clusters#deleteamaascluster). After 15 minutes, a **Force Delete Cluster** option is available in the **Settings drop-down menu**. The drop-down menu will provide you with an estimated remaining duration before the force deletion becomes available.
+
+<br />
+
+<WarningBox>
+A force delete can result in Palette-provisioned resources being missed during the removal process. Verify there are no remaining resources. Failure to remove provisioned resources can result in unexpected costs.
```

Comment:
"Verify there are no remaining resources" should be clarified to include reference to MAAS.

Maybe, "Verify that any MAAS machines associated with the cluster have been released"